Assets/Scripts/Game/EnemySenses.cs

```diff
@@ -728,8 +728,8 @@ void GetTargets()
                     if (targetBehaviour.EntityType == EntityTypes.EnemyMonster || targetBehaviour.EntityType == EntityTypes.EnemyClass)
                         targetSenses = targetBehaviour.GetComponent<EnemySenses>();
 
-                    // For now, quest AI can't be targeted
-                    if (targetSenses && targetSenses.QuestBehaviour)
+                    // Allows infighting via custom quest action
+                    if (targetSenses && targetSenses.QuestBehaviour && !targetSenses.QuestBehaviour.IsAttackableByAI)
```

Comment:
IsAttackableByAI isn't a member of QuestResourceBehaviour. The target field will also need to be added before it can be referenced.

I'm happy to add this down the track when I merge this one though. :)

Reply:
Sorry, I'm just the intermediary... Thanks for understanding Interkarma.